.github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml

```
@@ -0,0 +1,46 @@
+name: "💭 Rest-Api Feature Request"
```

```suggestion
name: "💭 Rest-API Feature Request"
```
Check your capitalization throughout.